workflow/Snakefile

```diff
@@ -121,20 +139,55 @@ rule samtools_index:
 	conda:
 			"envs/samtools.yaml"
 	shell:
-        	"samtools index {input}"
+        	"samtools index {input} -o {output}"
 rule sniffles:
 	input:
-			bam="results/sorted/{sample}.sorted.bam",
-			bai="results/samtools/bai/{sample}.sorted.bam.bai"
+			bam="results/sorted/patient.sorted.bam",
+			bai="results/sorted/patient.sorted.bam.bai",
+			genome="resources/GRCh38.primary_assembly.genome.fa"
 	output:
-			"results/sniffles/{sample}.vcf"
+			"results/sniffles/patient_sniffles.vcf"
 	conda:
 			"envs/sniffles.yaml"
-	benchmark: "benchmarks/{sample}.tsv"
+	benchmark: "benchmarks/sniffles.tsv"
+	threads: 
+			8
+	resources: 
+			mem_mb= 32000,
+			runtime= 1440   
+	shell:
+			"sniffles --input {input.bam} --reference {input.genome} --vcf {output}"
+
+rule cutesv:
+	input:
+			bam="results/sorted/patient.sorted.bam",
+			gen="resources/GRCh38.primary_assembly.genome.fa",
+	output:
+			"results/cutesv/patient_cutesv.vcf"
+	conda:
+			"envs/cutesv.yaml"
+	benchmark: "benchmarks/cutesv.tsv"
+	threads: 
+			8
+	resources: 
+			mem_mb= 32000,
+			runtime= 1440   
+	shell:
+        	"cuteSV {input.bam} {input.gen} {output} results/"
+rule bcftools:
+	input:
+			sniffles="results/sniffles/patient_sniffles.vcf",
+			cutesv="results/cutesv/patient_cutesv.vcf",
+			bash="resources/variants_analysis.sh"
```

Comment:
Scripts should not be declared as inputs. If the script is simple enough it should be added directly into the `shell` directive. If it's more complex (which I suspect is the case) then you should replace `shell` with the `script` directive. Docs here: https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#bash